configuration/pih/htmlforms/hiv/section-pmtct-history.xml

```diff
@@ -260,6 +265,14 @@
                         <obs conceptId="CIEL:164400"/>
                     </p>
 
+                    <label>
+                        <uimessage code="pihcore.hiv.statusKnownBeforePregnancy"/>
+                    </label>
+                    <p class="radio">
+                        <obs conceptId="PIH:13955" answerConceptIds="PIH:13956, PIH:1066"
```

Comment:
To be consistent, why not use yes/1065 and no/1066?

This was suggested for known hiv status for contacts so the PIH:13955 had other answers.  It was decided not to ask/use that question for contacts.